### DISCUSSION TITLE
Can't connect wallet on mobile using Metamask

### DISCUSSION THREAD

**Discussion by mzspcgg** [rainbow-me/rainbowkit#1341] [Q&A] [answered]
Rainbowkit "Connect Wallet" button stopped working on both iOS and Android when you try to connect using MetaMask.

It also doesn't work when you click "Connect Wallet" on mobile on your official website: https://www.rainbowkit.com.

The behavior is exactly like in this thread: https://github.com/rainbow-me/rainbowkit/issues/746.

Note, it also didn't work for me using version 1.0.1.

**Comment by mzspcgg** [comment]
Any news with this bug?

**Reply by inna0202** [reply]
it doesn't work for me either

**Comment by DanielSinclair** [comment, marked as answer]
WalletConnect support was deprecated with WalletConnect v1, so you'll need to update to RainbowKit [1.0.5](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%401.0.5) or greater for WalletConnect v2 support. It is also recommended that you upgrade to the latest version of `wagmi` as the WalletConnect team continues to improve network stability, which is usually the root cause for the behavior you're seeing (i.e. requests are blocked behind a VPN or in some regions). Don't forget to read the guide [here](https://www.rainbowkit.com/guides/walletconnect-v2) and provide a valid `projectId`. Let me know if you see see any other issues with the MetaMask connector.
